lumicks/pylake/population/detail/mixin.py

```diff
@@ -151,3 +174,10 @@ def plot_path(self, trace, *, trace_kwargs=None, path_kwargs=None):
         path_kwargs = {"c": "tab:blue", "lw": 2, **(path_kwargs or {})}
         emission_path = self.emission_path(trace)
         emission_path.plot(**path_kwargs)
+
+
+@dataclass(frozen=True)
+class BaseModel:
```

Comment:
I feel the name `BaseModel` is a bit too general/nondescript. I'm afraid that might bite us in the future. Thinking what GMMs and HMMs have in common, maybe `LatentVariableModel` or `TimeSeriesModel`?

What do you think?